packages/core/package.json

```
@@ -57,7 +57,8 @@
   "devDependencies": {
     "@medplum/definitions": "*",
     "@medplum/fhirtypes": "*",
-    "jest-websocket-mock": "2.5.0"
+    "jest-websocket-mock": "2.5.0",
+    "web-streams-polyfill": "^3.2.1"
```

```suggestion
    "web-streams-polyfill": "3.2.1"
```
Oh yeah we like to use exact versions here, you may have to rerun `npm install` again after this change.